Bottom: 7ae07701e0e35c8189e821e638ad4902c97e88e7
Top:    f2bbd90e42ffecc5526a8a89eee1ea85006ba567
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-15 02:30:39 +0000

Refresh of drop-in-pcb-gl-code-various-me

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index be9c633..6f38fb8 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -547,13 +547,12 @@ ghid_set_layer (const char *name, int group, int empty)
 	  return 0;
 	case SL_ASSY:
 	  return 0;
-	case SL_RATS:
-	  gport->trans_lines = TRUE;
-	  return 1;
 	case SL_PDRILL:
 	case SL_UDRILL:
 	  return 1;
 	case SL_RATS:
+	  if (PCB->RatOn)
+	    gport->trans_lines = TRUE;
 	  return PCB->RatOn;
 	}
     }
